modules/registrars/synergywholesaledomains/synergywholesaledomains.php

```diff
@@ -610,9 +622,37 @@ function synergywholesaledomains_RegisterDomain(array $params)
         $request['costPrice'] = $params['premiumCost'];
         $request['premium'] = true;
     }
-    
+
     try {
         synergywholesaledomains_apiRequest('domainRegister', $params, $request);
+
+        // If defaultDnsConfig is set
+        if (!empty($params["defaultDnsConfig"])) {
+            // If defaultDnsConfig is Parked, FreeDns or Forwarding
+            if (in_array($params["defaultDnsConfig"], ['2', '3', '4'])) {
+                synergywholesaledomains_apiRequest('updateNameServers', $params, [
+                    'domainName' => $params['domainName'],
+                    'dnsConfigType' => $params["defaultDnsConfig"],
+                    'nameServers' => $dnsHostingNameservers,
+                ], false);
+            }
+
+            // If defaultDnsConfig is FreeDns or Forwarding
+            if (in_array($params["defaultDnsConfig"], ['2', '4'])) {
+                Capsule::table('tbldomains')
+                    ->where('id', $params['domainid'])
+                    ->update(['dnsmanagement' => 1]);
+            }
+
+            // If defaultDnsConfig is customNS
+            if ($params["defaultDnsConfig"] == '1') {
```

Comment:
Please use single quotes for the array access rather than double quotes, unless you need to format a string.